docs/modules/ROOT/pages/api-reference/neo4jgraphql.adoc

```diff
@@ -90,12 +90,18 @@ Accepts all of the options from https://github.com/ardatan/graphql-tools/blob/%4
 |===
 |Name and Type |Description
 
-|`secret` +
+|`jwksEndpoint` +
  +
  Type: `string`
-|The secret used to encode JWT tokens. +
+|The OpenID Public Key Set Endpoint used to verify JWT tokens. The JWKS used to store the verification information is typically stored at an endpoint similar to \https://YOUR_DOMAIN/.well-known/jwks.json. +
++
+*Recommended*
+
+|`secret` +
  +
- *Required* unless passing in decoded tokens.
+ Type: `string`
+|The secret used to encode JWT tokens.
+|===
 
```

Comment:
If you take a look at the preview site commented by the bot, you'll see that you've broken the table a bit here. You _should_ be able to apply the suggestion below to fix this. I've also removed "recommended" as I'm of the opinion that an API reference should be purely factual and not opinionated.

```suggestion
|`jwksEndpoint` +
 +
 Type: `string`
|The OpenID Public Key Set Endpoint used to verify JWT tokens. The JWKS used to store the verification information is typically stored at an endpoint similar to \https://YOUR_DOMAIN/.well-known/jwks.json.

|`secret` +
 +
 Type: `string`
|The secret used to encode JWT tokens.

```